package/yast2-bootloader.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Jun  4 15:08:17 UTC 2018 - jreidinger@suse.com
+
+- Propose fips boot parameter if it is used for installation
+  on s390 (bsc#1043029)
```

Comment:
in fact you backport two fixes, so please mention both in changes ( another one have different bsc ).